main/test/flix/Test.Handler.NewObject.flix

```diff
@@ -30,9 +30,9 @@ mod Test.Handler.NewObject {
 
     def newGenAskRunnable(r: Ref[Int32, rc]): Runnable \ IO = new Runnable {
         def $run(_this: Runnable): Unit \ { rc } =
-            try {
+            run {
                 Ask.ask(
-                    try {
+                    run {
```

Comment:
Weird, parsing does not allow nested run-with

Reply:
Uh oh. You may have to check my stupid code!

Reply:
I will revert these to errors to `try-with` for now and fix it when we also remove the old `try-with` syntax completely (it's on 2 places in the entire code base that needs to be updated)

Reply:
Can we not fix it now? Separately?

Reply:
Its seems like a pretty bad bug, so would be good to have a fix in.

Reply:
True I have been working on it for 45 minutes now at it is not straightforward why this happens

Reply:
Gosh.